src/core_ocean/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -239,12 +246,22 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, layerThickness, layerThicknessEdg
                              /(zMid(k, cell1) - zMid(k + 1, cell1)) &
                              + slopeTriadDown(k, 2, iEdge)*(tracers(iTr, k, cell2) - tracers(iTr, k + 1, cell2)) &
                              /(zMid(k, cell2) - zMid(k + 1, cell2)))
+               if (minLevelCell(cell1) < minLevelEdgeBot(iEdge)) then
+                  flux_term2 = flux_term2 + coef*RediKappa(iEdge)* &
+                               layerThicknessEdge(k, iEdge)*0.25_RKIND*kappaRediEdgeVal*slopeTriadUp(k, 1, iEdge)* &
+                               (tracers(iTr, k - 1, cell1) - tracers(iTr, k, cell1))/(zMid(k - 1, cell1) - zMid(k, cell1))
+               endif
+               if (minLevelCell(cell2) < minLevelEdgeBot(iEdge)) then
+                  flux_term2 = flux_term2 + coef*RediKappa(iEdge)* &
+                               layerThicknessEdge(k, iEdge)*0.25_RKIND*kappaRediEdgeVal*slopeTriadUp(k, 2, iEdge)* &
+                               (tracers(iTr, k - 1, cell2) - tracers(iTr, k, cell2))/(zMid(k - 1, cell2) - zMid(k, cell2))
+               endif
```

Comment:
@mark-petersen (or maybe @vanroekel) could you take a careful look at these Redi changes, mostly because there among the more complex in this PR and I don't know Redi well enough to check them?

Reply:
Yes. I have a convergence test and idealized Redi test I can rerun.

Reply:
Thanks @mark-petersen. So far I have only run the `Redi_verification/SouthernOceanSlice40` test case.

Reply:
I think this will be fine for `minLevelCell = 1`, but not for other cases. I'll try make suggestions what I think it should be above.  Also I believe `k=1` at this point, so this will give out of bounds errors if this if is triggered.